orchestra/static/orchestra/dashboard/tasktable.html

```diff
@@ -31,23 +31,28 @@ <h3>
     </div>
     <div class="row">
           <table ng-if="vm.tasktable.tasks.length > 0"
-                 st-table="vm.tasktable.tasks"
+                 st-table="displayedTasks"
+                 st-safe-src="vm.tasktable.tasks"
                  class="table table-striped">
             <thead>
               <tr>
-                <th>Priority</th>
-                <th>Project</th>
-                <th>Task</th>
-                <th>Details</th>
-                <th>Next steps</th>
-                <th>Start by</th>
-                <th>Due on</th>
+                <th colspan="1"><input st-search="" class="form-control" placeholder="Search table" type="text"/></th>
```

Comment:
Whoops I meant to share Meredith's wireframes for this functionality. Can we move this to the right of the "Active" label?

Link to wireframe (the Abstract password is on LastPass): https://app.goabstract.com/projects/71af41c0-8b6c-11e8-95bd-818d6220cb4d/branches/master/commits/5932abee5247c44a5e832f7f102a47b0e0e3091d/files/16053EDA-E882-469E-966E-626370222516/layers/4C9E8259-1546-4578-A077-982C61343A9C?mode=design

Obviously Orchestra doesn't have the look and feel of the wireframes yet, but it'll give you an idea for placement :)